jdk/test/sun/security/pkcs11/nss/p11-nss-sensitive.txt

```diff
@@ -0,0 +1,58 @@
+
+# Configuration to run unit tests with NSS
+# Marks private and secret keys as sensitive
+
+name = NSS
+
+slot = 1
+
+#showInfo = true
+
+library = ${pkcs11test.nss.lib}
+
+nssArgs = "configdir='${pkcs11test.nss.db}' certPrefix='' keyPrefix='' secmod='secmod.db' flags=readOnly"
+
+disabledMechanisms = {
+  CKM_DSA_SHA224
+  CKM_DSA_SHA256
+  CKM_DSA_SHA384
+  CKM_DSA_SHA512
+  CKM_DSA_SHA3_224
+  CKM_DSA_SHA3_256
+  CKM_DSA_SHA3_384
+  CKM_DSA_SHA3_512
```

Comment:
These `*_SHA3_*` entries here and below have been removed from the 11u-dev downport because SHA3 support was only [introduced in JDK 16](https://bugs.openjdk.org/browse/JDK-8242332) (see [the 11u review thread](https://mail.openjdk.org/pipermail/jdk-updates-dev/2021-May/006096.html)).

I suggest to remove them here as well to be on par with the 11u downport.

Reply:
Done.

Reply:
They should be kept. The definitions are added by JDK-8244154, not JDK-8242332. You said the tests passed before they were removed, right?

Reply:
I've added them back.